RELEASE-NOTES.txt

```diff
@@ -1,5 +1,6 @@
 Unreleased
 ---
+* [***] Enable VideoPress block on iOS [#5692]
```

Comment:
I'm wondering if we should specify that this is only for simple WordPress.com sites? I know this limitation tripped a couple of people up when testing internally, so it could be worth clarifying for any users who might check out the release notes.

Reply:
Good point, I'll expand the entry specifying that it's enabled on iOS and only for simple WP sites 👍 . Similarly, I'll also update the release notes in https://github.com/wordpress-mobile/WordPress-iOS/pull/20580.

Reply:
I've updated the release notes in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5692/commits/30af943480f8bdd461356ea926f37a2c0973d1fd.